contracts/Registries/Overrides.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.11;
+
+import "../Utilities/Ownable.sol";
+
+interface ICurveRegistry {
+    function get_pool_from_lp_token(address) external view returns (address);
+}
+
+interface ICurveAddressesProvider {
+    function max_id() external view returns (uint256);
+
+    function get_address(uint256) external view returns (address);
+}
+
+contract CurveRegistryOverrides is Ownable {
+    ICurveAddressesProvider public curveAddressesProvider;
+    mapping(address => address) public poolByLpOverride;
+
+    constructor(address _curveAddressesProviderAddress) {
+        require(
+            _curveAddressesProviderAddress != address(0),
+            "Missing Curve Addresses Provider address"
+        );
+        curveAddressesProvider = ICurveAddressesProvider(
+            _curveAddressesProviderAddress
+        );
+    }
+
+    /// @notice Returns all curve registries
```

Comment:
Don't think all addresses provided by the address provider are necessarily registries, not really an issue but just don't want it to be misleading

Reply:
updated function to return all curve registry addresses that are currently set